src/components/SUE/report/SueReportLocation.tsx

```diff
@@ -108,7 +107,13 @@ export const SueReportLocation = ({
   const systemPermission = useSystemPermission();
   const { appDesignSystem = {} } = useContext(ConfigurationsContext);
   const { sueStatus = {} } = appDesignSystem;
-  const { statusBorderColors = {}, statusTextColors = {}, statusViewColors = {} } = sueStatus;
+  const { mapPinColors = {} } = sueStatus;
+  const {
+    activeBackgroundColor = {},
+    activeIconColor = {},
+    backgroundColor = {},
+    iconColor = {}
```

Comment:
ääh.. i guess these are string values? so fallbacks to empty objects seem wrong to me

Reply:
ok i see, these are object actually with different values. so please make these objects plural names, like `statusBorderColors`, as it holds the same keys/values, right!?

```
"statusBorderColors": {
  "Wird geprüft": "#93140B",
  "In Bearbeitung": "#853D14",
  "Abgeschlossen": "#10603F",
  "disabled": "#707070"
},
...
"backgroundColor": {
  "Wird geprüft": "#93140B",
  "In Bearbeitung": "#BE7D04",
  "Abgeschlossen": "#10603F"
},
```

Reply:
Okay, I updated the nomenclature. 👍